extensions/azure-eventhubs/deployment/pom.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one or more
+    contributor license agreements.  See the NOTICE file distributed with
+    this work for additional information regarding copyright ownership.
+    The ASF licenses this file to You under the Apache License, Version 2.0
+    (the "License"); you may not use this file except in compliance with
+    the License.  You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.apache.camel.quarkus</groupId>
+        <artifactId>camel-quarkus-azure-eventhubs-parent</artifactId>
+        <version>1.7.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>camel-quarkus-azure-eventhubs-deployment</artifactId>
+    <name>Camel Quarkus :: Azure Event Hubs :: Deployment</name>
+
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.amqphub.quarkus</groupId>
+                <artifactId>quarkus-qpid-jms-deployment</artifactId>
+                <version>${quarkus-qpid-jms.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+            <!-- quarkus-qpid-jms-deployment is currently not part quarkus-qpid-jms-deployment-->
```

Comment:
Sorry, I don't get this one

Reply:
It is copied from `camel-quarkus-qpid` (by @jamesnetherton IIRC) and I think it originally intended to say that quarkus-qpid-jms-deployment itself was not managed in quarkus-qpid-jms-deployment. It would be worth checking whether it is still the case.

Reply:
Actually, importing quarkus-qpid-jms-deployment it is perhaps not necessary anymore. In some phase I experimented with depending on quarkus-qpid-jms to solve some problem that turned out to be solvable differently. Let me check if it can be removed.

Reply:
Yes, it can be removed.

Reply:
Thanks for noticing

Reply:
Thanks to you Peter, I had no idea it should be removed but just find it strange :)